docs/pages/admin-guides/deploy-a-cluster/deployments/gcp.mdx

```diff
@@ -117,6 +117,10 @@ storage:
 Load Balancing is required for Proxy and SSH traffic. Use `TCP Load Balancing` as
 Teleport requires custom ports for SSH and Web Traffic.
 
+GCP sets a default Load Balancer timeout of 30 seconds. You should either increase this to be longer than the Teleport Auth service default keepalive interval of 300 seconds or decrease the Teleport ```keep_alive_interval``` to be lower than the GCP timeout value.
```

Comment:
```suggestion
GCP sets a default Load Balancer timeout of 30 seconds. You should either increase this to be longer than the Teleport Auth Service default keepalive interval of 300 seconds or decrease the Teleport `keep_alive_interval` to be lower than the GCP timeout value.
```

Reply:
I would also be more explicit that `keep_alive_interval` is a configuration field, e.g., indicating that it's a field in the configuration file and specifying which section in the file the field belongs to.

Reply:
It is better if they don't decrease this below 30 seconds because it will cause a high volume of logs so if we can be more opinionated with the language i'd prefer to recommend increasing the LB timeout.